src/Core/VsVersion/IVsVersion.cs

```diff
@@ -36,5 +36,10 @@ public interface IVsVersion
         /// VS display version, as seen in the About window. Example: "16.9.0 Preview 3.0"
         /// </summary>
         string DisplayVersion { get; }
+
+        /// <summary>
+        /// Major VS build version. Example: for "16.9.30914.41" it'll be "16"
+        /// </summary>
+        string MajorInstallationVersion { get; }
```

Comment:
I would have made the changes to the this interface first and pushed them in a separate PR. It's completely self-contained, would have been quick to do and review, and would have increased the amount of "done" work.